src/batched/dense/KokkosBatched_Scale_Decl.hpp

```diff
@@ -15,7 +14,12 @@ namespace KokkosBatched {
 struct SerialScale {
   template <typename ScalarType, typename AViewType>
   KOKKOS_INLINE_FUNCTION static int invoke(const ScalarType alpha,
```

Comment:
decorate with `[decprecated]`?

Reply:
Thanks! Decorated `Set` on #1448 and `Scale` on #1454.